src/sweetalert2.js

```diff
@@ -1002,7 +1006,7 @@ const sweetAlert = (...args) => {
       }
     }
 
-    openModal(params.animation, params.onOpen)
+    openModal(params.animation, params.onOpen, params.onBeforeOpen)
```

Comment:
Nit: it would be better if the `onBeforeOpen` will be prior to `onOpen`:

```js
openModal(params.animation, params.onBeforeOpen, params.onOpen)
```